Redesigned go back button

elinhagman opened the pull request.

Turbobus commented:
The button in it self looks good and final. I would however want it to be smaller as for example in this picture the button covers the player fox. I would also add a bit of margin near the screen edge so it would not be exactly up agains it both on top and to the left

![image](https://user-images.githubusercontent.com/59963570/169146065-4912c96f-e46b-4c8d-8eeb-f742f9522fa3.png)

elinhagman commented:
I made the button a little smaller now and changed its color to fit the other buttons in the game better. I noticed that when i merged main into the branch the question mark button had been moved to behind the home button. I moved it back to the side of the home button again. Let me know if this is not correct!

Elin1043 commented:
Looks good!
